packages/http/README.md

```diff
@@ -1,152 +1,148 @@
 # Prism HTTP
 
-This package provides HTTP APIs for forwarding and mocking requests.
-
-## HTTP Forwarder
-
-> Note: The current API is still experimental - changes can be introduced in future.
-
-Module used to forward provided request. Is exposes single `forward` method and can be initialized with `IHttpOperation` and `IHttpRequest`.
+This package provides a http client featuring the ability to:
+
+- mock requests instead of hitting the server
+- validate the request and the response according to an openapi spec
+
+# Basic Usages
+
+> Note: examples in this document use the following spec
+
+```yaml
+---
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      parameters:
+        - name: title
+          in: query
+          style: form
+          schema:
+            enum:
+              - eat
+              - drink
+      responses:
+        200:
+          description: Get Todo Items
+```
 
-### Forwarder props
+## I want to use Prism to mock responses
 
-Object containing:
+[Try it!](https://repl.it/@ChrisMiaskowski/prism-http-client-basic-mocking);
 
-- `resource` containing `IHttpOperation` _(optional)_
-- `input` `IHttpRequest` wrapped with `IPrismInput`
-- `timeout` overrides default timeout _(optional)_
-- `cancelToken` cancellation token _(optional)_
+```javascript
+const Prism = require('@stoplight/prism-http');
+const path = require('path');
 
-Properties used from `resource: IHttpOperation`:
+// Create Prism instance and configure it as a mocker generating static examples
+const config = { mock: { dynamic: false } };
+const prism = Prism.createInstance(config);
 
-- `servers[]` - if not empty the first element will be used as server url
+// Load openapi spec file
+const specPath = path.resolve(process.cwd(), 'basic.oas3.yaml');
+prism
+  .load({ path: specPath })
+  .then(() => {
+    // Make a "GET /todos" request
+    return prism.process({
+      method: 'get',
+      url: {
+        path: '/todos',
+      },
+    });
+  })
+  .then(prismResponse => {
+    console.log(prismResponse.output);
+  });
+```
 
-The type of `input` parameter `IPrismInput<IHttpRequest>` is defined as:
+Output
 
-```ts
-interface IPrismInput<IHttpRequest> {
-  data: IHttpRequest;
-  validations: {
-    input: IPrismDiagnostic[];
+```bash
+{ statusCode: 200,
+  headers: { 'Content-type': 'text/plain' },
+  body: undefined }
 ```
 
-Validations are not supported at the moment for Http Forwarder.
+## I want Prism to make a http request a server
```

Comment:
This might not be clear enough for people without context. We want to explain that Prism HTTP Client is a generic HTTP client first, and it also happens to have mock mode enabled. 

Imagine we are releasing a axios competitor, this README https://www.npmjs.com/package/axios should be an inspiration. 

One of the features of our HTTP client is that it can shortcut and return mocks instead. 

Instead of talking to localhost:4010 which might confuse people and make them think they are talking to Prism HTTP Server, why don't we make calls to https://pokeapi.co/ or api.example.org or literally anything other than localhost.

Reply:
> Instead of talking to localhost:4010 which might confuse people and make them think they are talking to Prism HTTP Server, why don't we make calls to https://pokeapi.co/ or api.example.org or literally anything other than localhost.

This is exactly what I did :) I tested it using the prism CLI

We could use api.example.org or pokeapi.co buuuuut api.example.org does not return stuff... pokeapi.co doesn't do anything relevant either.

Instead, if someone runs the Prism CLI with the spec provided at the top of the documentation they will be able to actually test this stuff.

Reply:
Ok, I like the idea that this should definitely actually run. Pokeapi is an API with JSON data so if you take a second look you'll see you can construcvt URLs.

https://pokeapi.co/api/v2/pokemon/ditto/

JSON! 

I respect that using the CLI is a good way to have a server running locally that we know they have, but can you see how it is confusing for the user? Using prism to talk to prism is meta and conflates two different components and their use.

Reply:
Gotcha! Let's GO with Pokemons! ;)

Reply:
Actually, I picked cats :P